src/js/helpers/Util.js

```diff
@@ -154,6 +157,34 @@ var Util = {
   },
   getUniqueId: function () {
     return ++consecutiveNumber;
+  },
+  detectPathsInObject: function (obj, startKey, excludePaths = []) {
```

Comment:
Pretty neat!

Because we later use these paths with the `objectPath` library, would it make sense to call this function something like `convertToObjectPath` to highlight the dependency on the format expected by `objectPath.get()`?